src/System.Text.Encodings.Web/src/System/Text/Encodings/Web/CodePointFilter.cs

```diff
@@ -49,46 +50,46 @@ public CodePointFilter(params UnicodeRange[] allowedRanges)
         }
 
         /// <summary>
-        /// Allows the character specified by <paramref name="c"/> through the filter.
+        /// Allows the character specified by <paramref name="character"/> through the filter.
         /// </summary>
         /// <returns>
         /// The 'this' instance.
         /// </returns>
-        public CodePointFilter AllowChar(char c)
+        public CodePointFilter AllowChar(char character)
         {
-            _allowedCharsBitmap.AllowCharacter(c);
+            _allowedCharsBitmap.AllowCharacter(character);
             return this;
         }
 
         /// <summary>
-        /// Allows all characters specified by <paramref name="chars"/> through the filter.
+        /// Allows all characters specified by <paramref name="characters"/> through the filter.
         /// </summary>
         /// <returns>
         /// The 'this' instance.
         /// </returns>
-        public CodePointFilter AllowChars(params char[] chars)
+        public CodePointFilter AllowChars(params char[] characters)
         {
-            if (chars != null)
+            if (characters != null)
```

Comment:
I agree that it would be good to look at these APIs and clean that up. But I propose I do it as a separate workitem. The goal of this PR is to generalize the abstraction APIs, and implement the same behavior like in current ASP.NET implementation. Cleaning up null behavior seems like a totally separate and potentially large workitem.